src/components/firebase/Firebase.js

```diff
@@ -128,12 +146,76 @@ function getProFiLabelTexts() {
   }
 }
 
-function getProFiTexts(){
+function getProjectsFiTexts(){
   return {
     label: getProFiLabelTexts(),
     button: getProFiButtonTexts()
   }
 }
 
-export { getGenEnTexts, getGenFiTexts, getFrontpageLabelTexts, 
-        getProEnTexts, getProFiTexts };
\ No newline at end of file
+
+function getBackFiLabelTexts() {
+  if(backFiLTextSnapshot.exists){
+    return backFiLTextSnapshot.data();
+  }
+}
+
+function getBackFiEducationTexts() {
+  const doc_array = {
+    I: backFiEdTextSnapshot.docs[0].data(),
+    II: backFiEdTextSnapshot.docs[1].data()
+  };
+  return doc_array;
+}
+
+function getBackFiExperienceTexts() {
+  const doc_array = {
+    I: backFiExTextSnapshot.docs[0].data(),
+    II: backFiExTextSnapshot.docs[1].data()
+  };
+  return doc_array;
+}
+
+function getBackgroundFiTexts(){
+  return {
+    label: getBackFiLabelTexts(),
+    education: getBackFiEducationTexts(),
+    experience: getBackFiExperienceTexts()
+  }
+}
+
+
+function getBackEnLabelTexts() {
+  if(backEnLTextSnapshot.exists){
+    return backEnLTextSnapshot.data();
+  }
+}
+
+function getBackEnEducationTexts() {
+  const doc_array = {
+    I: backEnEdTextSnapshot.docs[0].data(),
```

Comment:
Although now there is only two education info packets, should maybe do this in a loop. Same with experience packets.